src/mcp_server_helm/commands/basic.py

```
@@ -0,0 +1,64 @@
+def helm_verify(path: str, keyring: Optional[str] = None) -> str:
+    """
+    Verifies that a chart at the given path has been signed and is valid.
+    """
+    logger.info(f"Running helm verify with path={path}")
+
+    cmd = ["helm", "verify", path]
+
+    if keyring:
+        cmd.extend(["--keyring", keyring])
+
+    return execute_helm_command(cmd)
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)